pytradfri/gateway.py

```diff
@@ -65,6 +67,17 @@ def _get_mood_parent(self):
         """Get the parent of all moods."""
         return self.api('get', [ROOT_MOODS])[0]
 
+    def get_smart_tasks(self):
+        """Return the transitions linked to the gateway."""
+        tasks = self.api('get', [ROOT_SMART_TASKS])
+
+        return [self.get_task(task) for task in tasks]
+
+    def get_task(self, task_id):
```

Comment:
We should be consistent in the naming, this one should either be called `get_smart_task` or the other one be `get_tasks`